spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -1,10 +1,15 @@
 import {
```

Comment:
Side thought: I keep getting torn between if we should have a lot of this logic in actions or middleware. Like have middleware that essentially sets up pubsub to RCS but keep the logic in RCS; do not use middleware to fire additional actions from one action but rather do like connect() on an explicit CONNECT action type.. I think the two implementations would actually look very similar but the dependency relationships might look

Reply:
yeah, I guess we could do that